lib/pbench/test/unit/server/auth/test_auth.py

```diff
@@ -298,24 +298,29 @@ def test_wait_for_oidc_server_fail(self, make_logger):
         with pytest.raises(OpenIDClient.NotConfigured):
             OpenIDClient.wait_for_oidc_server(config, make_logger)
 
-        section = {"server_url": "https://example.com"}
+        section = {
+            "server_url": "https://example.com",
+            "realm": "realm",
+            "cert_location": "/ca.crt",
+        }
         config["openid"] = section
 
-        # Keycloak health returning response but status is not UP
+        # Keycloak well-known endpoint returning response with valid issuer
```

Comment:
I suspect that "valid issuer" is not the correct comment here. 😆

Reply:
Yeah probably just an issuer.